src/index.ts

```diff
@@ -113,23 +113,24 @@ export type KycStatusResponse = {
 / Fiat Account Endpoint Types
 */
 
-// Path parameters for POST /accounts/:fiatAccountSchema
-export type AddFiatAccountRequestParams = {
+// Request body for POST /accounts/
+export type PostFiatAccountRequestBody = {
```

Comment:
Renamed Add to Get to stay consistent with http verb like other accounts endpoints